models/accounts.js

```diff
@@ -0,0 +1,83 @@
+const mongoose = require('mongoose');
+
+const accountSchema = new mongoose.Schema({
+	title: {
+		type: String,
+		required: true,
+		unique: true
+	},
+	currency: {
+		type: String,
+		required: true
+	},
+	ammount: {
+		type: Number,
+		default: 0
+	},
+	income: {
+		type: [mongoose.ObjectId],
+		ref: 'Income',
+		default: []
+	},
+	expence: {
+		type: [mongoose.ObjectId],
+		ref: 'Expence',
+		default: []
+	}
+}, {
+	timestamps: true
+});
+
+userSchema.methods.edit = function(title, currency) {
+	console.log(`Hi, I'm ${this.name}`);
+}
+
+userSchema.statics.register = async function(user) {
+	const temp = {
+		email: user.email,
+		password: bcrypt.hashSync(user.password, 10),
+		role: user.role,
+		credentials: {
```

Comment:
Why do you need field 'credentials'? Why dont store first_name, last_name and so on in root?